### PR TITLE
Document and modify accessibility vocabulary

### LINKED ISSUE #18: Accessibility Vocabulary
Currently, the [accessibility vocabulary](https://github.com/openeduhub/oeh-metadata-vocabs/blob/master/accessibilitySummary.ttl) holds 5 different choices: 

 - A
 - AA
 - AAA
 - WCAG
 - BITV

Unfortunately, I was not able to find any documentation about when exactly use which choice. Reading about [BITV](https://www.bitvtest.de/bitv_test.html) and WCAG, I found that BITV seems to be a superset of WCAG, i.e. if a site is BITV compliant it would also have a AAA WCAG rating. This makes the current definition of the value space slightly confusing:

- Why is there a WCAG value?
- There is no value for "not even A", i.e. a site that does not fulfill even the lowest standards,
- We could change to (in ascending order):
  - B (or some other name, no compliance)
  - A (WCAG)
  - AA (WCAG)
  - AAA (WCAG)
  - BITV

IMHO, it is very important to distinguish the scenario, where a rating is not available (missing meta-datum) and where a site is not compliant with any rating (currently also a missing meta-datum), so we should definitively add a "B" rating value.

### PR DESCRIPTION
**Pull request by MRuecklCC**
Consider this a proposal to resolve #18 please :) 

Changes:
 - remove the `none` and `WCAG` concept.
 - added descriptions
 - added `narrower` relations between concepts to indicate ordering
 - split `bitv` concept into `bitv-I` and `bitv-II`

Discussion highly appreciated!

### CODE REVIEW

File: accessibilitySummary.ttl [hunk 1]
```diff
@@ -6,40 +6,52 @@
...
-    skos:hasTopConcept <none>, <a>, <aa>, <aaa>, <bitv>, <wcag> .
+    skos:hasTopConcept <b>, <a>, <aa>, <aaa>, <bitv-I>, <bitv-II> .
 
-<none> a skos:Concept ;
-    skos:prefLabel "Nicht geprüft"@de ;
-    skos:prefLabel "Not tested"@en ;
+<b> a skos:Concept ;
+    skos:prefLabel "B" ;
+    skos:definition "The content does not comply to the WCAG A rating requirements."@en ;
+    skos:definition "Der Inhalt erfüllt nicht die WCAG Anforderungen für eine A-Wertung."@en ;
```
Comment (lummerland) [inline accessibilitySummary.ttl:20]: "@de"

File: accessibilitySummary.ttl [hunk 2]
```diff
@@ -6,40 +6,52 @@
...
+    dct:modified "2022-07-19"^^xsd:date ;
     dct:issued "2020-11-16"^^xsd:date ;
     dct:license <http://creativecommons.org/publicdomain/zero/1.0/> ;
-    skos:hasTopConcept <none>, <a>, <aa>, <aaa>, <bitv>, <wcag> .
+    skos:hasTopConcept <b>, <a>, <aa>, <aaa>, <bitv-I>, <bitv-II> .
 
-<none> a skos:Concept ;
-    skos:prefLabel "Nicht geprüft"@de ;
-    skos:prefLabel "Not tested"@en ;
+<b> a skos:Concept ;
```
Comment (lummerland) [inline accessibilitySummary.ttl:17]: Maybe instead of just "b" something more meaningful? E.g. "failed" :D

**Comment by sroertgen** [comment]
Regarding the change of identifiers I strongly suggest to talk to @torsten-simon 

The identifiers are just ...identifiers, so I'm not sure if we really need to change them. If changed, this might have implications on already assigned values and a migration has to be run.

Other than that I approve the description changes, just the language has to be adjusted as @lummerland suggested.

**Comment by torsten-simon** [comment]
If we change any current identifiers, I need a mapping from the old to the new id in order to map existing content. I'm not sure if "meaningful" identifiers are that important, afaik we use uuid identifiers for newer vocabs which might do the job just fine.

**Pull request closed without merging**